frontend/public/components/utils/units.js

```diff
@@ -408,4 +408,9 @@ export const formatBytesAsMiB = (bytes) => {
   return formatToFractionalDigits(mib, 1);
 };
 
+export const formatBytesAsGiBInteger = (bytes) => {
+  const gib = bytes / 1024 / 1024 / 1024;
+  return formatToFractionalDigits(gib, 0);
+};
```

Comment:
I'd suggest having one decimal place precision. Maybe

```suggestion
export const formatBytesAsGiB = (bytes) => {
  const gib = bytes / 1024 / 1024 / 1024;
  return formatToFractionalDigits(gib, 1);
};
```

This is also consistent with the `formatBytesAsMiB` helper.